impedance/models/circuits/fitting.py

```diff
@@ -355,7 +354,8 @@ def count_parens(string):
             eval_string += new
 
         if i == len(split) - 1:
-            eval_string += '])'
+            if len(split) > 1:  # do not add closing brackets if single element
+                eval_string += '])'
```

Comment:
nit: Could make this a single line: `if (i == len(split) - 1) and (len(split) > 1):`